src/input.service.ts

```diff
@@ -561,10 +560,7 @@ export class InputService {
    */
   private addKeyboardListeners() {
     this.subscriptions.push(
-      Observable.merge(
-        this.keyboardSrc,
-        Observable.fromEvent<KeyboardEvent>(window, 'keydown'),
-      ).subscribe(ev => {
+      merge(this.keyboardSrc, fromEvent<KeyboardEvent>(window, 'keydown')).subscribe(ev => {
```

Comment:
:'(

Reply:
hm? Just formatting?

Reply:
Just that it's adding to a subs array and managing observables that way for unsubbing. no big deal.

Reply:
ah, yea. For later refactors :)